modules/ob_common/manifests/params.pp

```diff
@@ -30,21 +30,41 @@
   $jdk_name     = 'jdk1.8.0_161'
   $java_home    = "${java_dir}/${jdk_name}"
 
-  $wso2_service_name  = "wso2${profile}"
   $version            = '2.0.0-ALPHA'
-  $pack               = "wso2-${profile}-${version}"
 
-  # Pack directories
```

Comment:
If you think this comment is not appropriate, please add a proper comment for the parameters defined here.

Reply:
I will add a more meaningful comment

Reply:
done